src/libANGLE/renderer/d3d/d3d11/ExternalImageSiblingImpl11.cpp

```diff
@@ -49,7 +49,7 @@ egl::Error ExternalImageSiblingImpl11::initialize(const egl::Display *display)
     mIsRenderable = (textureDesc.BindFlags & D3D11_BIND_RENDER_TARGET) &&
                     (resourceUsage & DXGI_USAGE_RENDER_TARGET_OUTPUT) &&
                     !(resourceUsage & DXGI_USAGE_READ_ONLY);
-
+    mIsDepthAttachment = (textureDesc.BindFlags & D3D11_BIND_DEPTH_STENCIL);
```

Comment:
nit: Wdyt of `mIsColorRenderable` and `mIsDepthRenderable`? (Which is pretty much in line with OpenGL terminology).

Reply:
https://github.com/shapr3d/angle/pull/20/commits/528ded27ea0852e405efd96cadf55c8a3ba956a4